engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -91,6 +92,7 @@ namespace dmGameSystem
     {
         dmObjectPool<SpriteComponent>   m_Components;
         dmArray<dmRender::RenderObject*> m_RenderObjects;
+        dmArray<float>                  m_BoundingVolumes;
```

Comment:
Opted for storing the bounding volume with the component type, as opposed to adding more data to the small RenderListEntry struct.